src/components/Type2020/Blueprint.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Blueprint for Q2 2020 Type transition to new font sizes and styles.
+ *
+ *
+ * Name proposals
+ * --------------------------
+ * Type refers to the current soon to be "legacy" Type components.
+ * We need something to differentiate from our current Type component while transitioning.
+ *   - Type2020 (working prototype in this doc & files)
+ *   - Type2020 / TwentyType
+ *   - TypeNew / NewType
+ *   - TrendyType
+ *   - ???
+ *
+ * Should we append the new name to the end of Type2020 components like below?
+ * TitleSmall vs TitleSmall2020
+ *
+ * Should we rename Type to TypeLegacy or similar?
+ *
+ *
+ * Problems to solve
+ * --------------------------
+ * Provide access for developers to the new Type styles while maintaining backwards compatability.
+ *
+ * Allow for a slow rollout to minimize the risk of things breaking.
+ *
+ * Make transitioning as seamless as possible, existing components should work without any developer
+ * interference aside from setting a new import name for the component.
+ * Ex: import Type2020.TitleXLarge as TitleXLarge
+ *
+ * Allow for easy deprecation of the legacy Type styles once all components are ready to 'upgrade'.
+ *
+ * Fix things we don't like about the current Type system and add things we wish we had.
+ *   - Less rigidity for passing extra properties like className
+ *   - ???
+ *
+ *
+ * Specifications from design
+ * --------------------------
+ *
+ * Spreadsheet:
+ * https://docs.google.com/spreadsheets/d/1LiKYSuW5Lfh24OjT8e0WgNLcxuAZrFlbzmw_NJjUNOo/edit?usp=sharing
+ *
+ * The spreadsheet has the naming of deprecated Type components like <TitleXLarge.Serif.Book500/>,
```

Comment:
I'd prefer not the `.Serif` part and would prefer a prop like `<Type2 letterform="serif|sans|mono"...` taking liberties with the or'ing, but `letterform="sans"`

Reply:
The one plus for dotted concatenation is it would give us the ability to directly copy/paste text styles out of Figma and into JSX components, if the designers setup their global styles in Figma to match the spreadsheet.

It's also what developers are already used to using as well, so this one will require a bit more weighing of pros/cons. I can see benefit for using props instead and a single `Type2` component as well.